CONTRIBUTING.md

```diff
@@ -64,6 +87,5 @@ reviewers, assign to yourself and add a label.
 - Typescript source code must be formatted according to
 [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html),
 as mentioned. There's a pipeline to ensure all of our code is good to go.
-- We try to use [conventional commits](https://www.conventionalcommits.org/)
-because it makes the process of generating changelogs way easier. So we encourage
-you to read at least the [summary](https://www.conventionalcommits.org/en/v1.0.0/#summary) and learn a bit about it.
\ No newline at end of file
+- We use [conventional commits](https://www.conventionalcommits.org/)
+because it makes the process of generating changelogs possible. If that's new for you, please take a moment to read it. You can start with the [summary](https://www.conventionalcommits.org/en/v1.0.0/#summary) and go from there.
```

Comment:
Optional: the conventional commits page doesn't go into the meaning of the _non-mandatory_ tags (i.e. the ones that aren't `feat` or `fix`). We could link [this page](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) (that is linked in the conventional commits page itself, albeit an earlier version of it) or just copy the list here.